js/src/dag.js

```diff
@@ -93,7 +94,10 @@ module.exports = (common) => {
         }, done)
       })
 
-      it('dag-cbor node with wrong multicodec', (done) => {
+      // This works because dag-pb will serialize any object. If the object
+      // has neither a `data` nor `links` field it's serialized as an empty
+      // object
+      it.skip('dag-cbor node with wrong multicodec', (done) => {
```

Comment:
Why skip?

Reply:
Sorry, that's a left-over, it should only skip in case of go-ipfs (there the comment is accurate).